packages/fxa-content-server/app/scripts/views/choose_what_to_sync.js

```diff
@@ -92,7 +92,10 @@ const View = FormView.extend({
     });
 
     return this.user.setAccount(account)
-      .then(this.onSubmitComplete);
+      .then(() => {
+        this.notifier.trigger('set-sync-engines', offeredSyncEngines.map(e => e.text));
```

Comment:
Do you need the .text here? `_getOfferedEngineIds` returns the element IDs rather than the text.

Reply:
Ah thanks, didn't realise. Hadn't actually got round to testing this one yet, was waiting for the other two to land. 😊